--- LOCAL/process_manager.py
+++ D-DRIVE/process_manager.py
@@ -109,11 +109,9 @@
             import ctypes
 
             mask = 1 << core
-            ctypes.windll.kernel32.SetProcessAffinityMask(
-
+            ctypes.windll.kernel32.SetProcessAffinityMask(  # type: ignore
                 ctypes.windll.kernel32.GetCurrentProcess(),
-                mask,
-
+                mask,  # type: ignore
             )
         else:
             os.sched_setaffinity(0, {core})
